index.src.html

```diff
@@ -17,96 +17,99 @@
     * User selects this prompt / notification.
     * User is shown a list of their example.com identities, e.g., "Sign in as Alice / Sign in as Bob."
-    * User picks an identity, is prompted for an authorization gesture (PIN, biometric, etc.) and provides this.
+    * User picks an identity, is prompted for an <a>authorization gesture</a> (PIN, biometric, etc.) and provides this.
 
-- Now, on the laptop:
-    * Web page shows that the selected user is signed in, and navigates to the signed-in page.
+- Now, back on the laptop:
+    * Web page shows that the selected user is signed-in, and navigates to the signed-in page.
 
 
-## Other configurations ## {#other-configurations}
+### Other use cases and configurations ### {#other-configurations}
 
 A variety of additional use cases and configurations are also possible, including (but not limited to):
 
-- User goes to example.com on their laptop, is guided through a flow to create and register a credential on their phone.
+- User navigates to example.com on their laptop, is guided through a flow to create and register a credential on their phone.
 
-- User employs a scoped credential as described above to authorize a single transaction, such as a payment or other financial
-    transaction.
+- A [RP] can cause a signed-in user to be prompted for their <a>authorization gesture</a> in order to authorize a single transaction, such as a payment or other financial transaction.
 
 
 # Conformance # {#conformance}
 
-This specification defines criteria for a <a>Conforming User Agent</a>. A User Agent MUST behave as described in this
-specification in order to be considered conformant. User Agents MAY implement algorithms given in this specification in any way
+This specification defines criteria for a <a>Conforming User Agent</a>: a User Agent MUST behave as described in this
+specification in order to be considered conformant. <a>Conforming User Agents</a> MAY implement algorithms given in this specification in any way
 desired, so long as the end result is indistinguishable from the result that would be obtained by the specification's
-algorithms. A conforming Web Authentication API User Agent MUST also be a conforming implementation of the IDL fragments of this
+algorithms. A conforming User Agent MUST also be a conforming implementation of the IDL fragments of this
 specification, as described in the “Web IDL” specification. [[!WebIDL-1]]
 
-This specification also defines a model of a Web Authentication compliant authenticator. This is a set of functional and
-security requirements for an authenticator to be usable by a User Agent that implements the Web Authentication API. As described
-in [[#use-cases]], the authenticator itself may be implemented in the operating system underlying the User Agent, or in external
+This specification also defines a model of a conformant Web Authentication authenticator (see [[#authenticator-model]]). This is a set of functional and
```

Comment:
Can we just say "Web Authenticator"? "Web Authentication authenticator" seems inelegant.